{"name": "Alert message and expected result should be correct.", "status": "passed", "steps": [{"name": "Login with invalid value. -> test_data:{'Description': 'Wrong password', 'Method': 'POST', 'Provider': 'native', 'Email': 'stone+02@test.com', 'Password': '1qazXSW@3edc', 'Http_status_code': '400', 'Error msg': 'Login Failed'}", "status": "passed", "start": 1677720271456, "stop": 1677720271904}, {"name": "Verify http_status code is as expected.", "status": "passed", "start": 1677720271904, "stop": 1677720271904}, {"name": "Verify error message is Login Failed.", "status": "passed", "start": 1677720271904, "stop": 1677720271904}], "attachments": [{"name": "log", "source": "405445c3-f136-4dc3-9d11-3b0ef60dfaec-attachment.txt", "type": "text/plain"}], "start": 1677720271455, "stop": 1677720271905, "uuid": "0a673326-714d-4d0a-9ca9-5a9b37a4d25d", "historyId": "fb1f981ff1a6eb255cf26f97dd119985", "testCaseId": "39a202243186ccfff89e85fdf8b5e5ba", "fullName": "test_api.test_user#test_login_failure", "labels": [{"name": "feature", "value": "User API"}, {"name": "story", "value": "Checkout login failure."}, {"name": "tag", "value": "UserAPI"}, {"name": "parentSuite", "value": "test_api"}, {"name": "suite", "value": "test_user"}, {"name": "host", "value": "Lins-MBP.GoMore-3F"}, {"name": "thread", "value": "42545-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_api.test_user"}]}